data/governments.txt

```diff
@@ -425,6 +425,12 @@ government "Hai"
 		"Hai (Unfettered Civilians)" .1
 		"Merchant" .01
 		"Elenctic Commune" .1
+	"penalty for"
+		assist -.15
+	"custom penalties for"
+		"Hai (Unfettered)"
+			capture 0
+			destroy 0
```

Comment:
This block can be ditched if we change the attitude towards `"Hai (Unfettered)"` from `-.1` to `-.01`